samtranslator/intrinsics/actions.py

```diff
@@ -333,11 +329,11 @@ def _handle_sub_action(self, input_dict, handler):  # type: ignore[no-untyped-de
         key = self.intrinsic_name
         sub_value = input_dict[key]
 
-        input_dict[key] = self._handle_sub_value(sub_value, handler)  # type: ignore[no-untyped-call]
+        input_dict[key] = self._handle_sub_value(sub_value, handler)
 
         return input_dict
 
-    def _handle_sub_value(self, sub_value, handler_method):  # type: ignore[no-untyped-def]
+    def _handle_sub_value(self, sub_value: Any, handler_method: Callable[[str, str], str]) -> Any:
```

Comment:
Is `sub_value` really `Any`? Maybe some `Union` will make sense?

Reply:
wait, why it is not `Any` here? We expect this to be anything right? See test `test_must_skip_invalid_input_dict()`

Reply:
Good catch.